### DISCUSSION TITLE
[Projects Beta]Column Actions

### DISCUSSION THREAD

**Discussion by macMikey** [community/community#17871] [Projects and Issues]
I'd like to be able to mass-apply a change to an entire column, e.g. closing all of them.

**Comment by ronnyroeller** [comment]
Same here.

Some other actions that would be required to use projects with many issues:
- Move many issues to another column
- Assign labels
- Set a project fields, e.g. we use one field for "epic" - and need assign many issues to a specific epic

**Comment by macMikey** [comment]
I think I'd also like to be able to mass-apply a milestone, too.
